src/modules/Exposure/Module.js

```diff
@@ -0,0 +1,56 @@
+/*
+* Changes the Image Exposure
+*/
+
+module.exports = function Brightness(options,UI){
+
+
+    var output;
+
+    function draw(input,callback,progressObj){
+
+        options.exposure = parseInt(options.exposure) || 0;
+        var exposure = Math.pow(2, options.exposure);
+        progressObj.stop(true);
+        progressObj.overrideFlag = true;
+
+        /*
+        In this case progress is handled by changepixel internally otherwise progressObj
+        needs to be overriden and used
+        For eg. progressObj = new SomeProgressModule()
```

Comment:
I don't think these comments are really necessary. They are included in the boilerplate so that the user can understand the boilerplate code.